src/relay/qnn/op/requantize.cc

```diff
@@ -263,6 +263,14 @@ bool RequantizeRel(const Array<Type>& types, int num_inputs, const Attrs& attrs,
     return false;
   }
 
+  if (types[0].as<IncompleteTypeNode>()) {
+    return false;
+  }
+  for (size_t i = 3; i < 5; ++i) {
```

Comment:
While we're adding a bunch of type checks, can you add a comment indicating what each input represents, something like `// Expected types: data, scale, zero_point, ...`

Reply:
Done